docs/linking/README.md

```diff
@@ -64,26 +65,31 @@ The end-goal of this phase is for the PISP Server to have a final list of DFSPs
 available and any relevant metadata about those DFPSs that are necessary to
 begin the linking process.
 
-![Pre-linking](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/mojaloop/pisp/master/docs/linking/0-pre-linking.puml)
+![Pre-linking](../out/linking/0-pre-linking.svg)
```

Comment:
I'm fine with leaving this as pre-linking for now for 2 reasons:
1. We already have a discovery step, for discovering the accounts to link
2. pre-linking can live outside the scope of linking, since it's an API call that can be cached, instead of on-demand by a user. I think the distinction may be helpful from our documentation etc.

Reply:
+1